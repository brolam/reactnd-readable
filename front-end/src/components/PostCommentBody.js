import React from 'react'

function PostCommentBody({ post }) {
  return (<div></div>)
}

export default PostCommentBody