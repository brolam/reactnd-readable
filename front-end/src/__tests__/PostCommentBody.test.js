import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import PostCommentBody from '../components/PostCommentBody'

test('render without error', () => {
  ReactDOM.render(
    <PostCommentBody post={postRedux} />, document.createElement('div'));
})

test('last Snapshot', () => {
  const postCommentBody = renderer.create(<PostCommentBody post={postRedux} />);
  let tree = postCommentBody.toJSON();
  expect(tree).toMatchSnapshot();
})


const postRedux = {
  category: "redux",
  id: "7xf0y6ziyjabvozdd253nd",
  title: "Udacity is the best place to learn Redux",
  timestamp: 1467166872634,
  author: "thingtwo",
  body: "Everyone says so after all.",
  deleted: false,
  voteScore: 5
}