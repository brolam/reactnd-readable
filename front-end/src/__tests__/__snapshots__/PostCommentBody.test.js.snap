// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`last Snapshot 1`] = `
<div
  className="post-page-body"
>
  <div
    className="post-page-author"
  >
    by 
    thingtwo
     
    a year ago
  </div>
  Everyone says so after all.
</div>
`;
